### DISCUSSION TITLE
Inverter fotovoltaico RIELLO PV RS 3.0

### DISCUSSION THREAD

**Discussion by Cupra1979** [indomus/forum#468] [Hardware & Firmware]
Buonasera a tutti!

Sto cercando da qualche tempo di integrare l'inverter del mio impianto fotovoltaico, un Riello mod. PV RS 3.0, che è dotato di connettività WiFi.

Sul manuale di istruzioni è riportata la seguente indicazione:
"_Gli inverter PV di questa serie sono provvisti di connessione Wi-Fi a doppio canale integrata. Utilizzando la APP dedicata “RS Connect”, è possibile stabilire una connessione locale al sistema utilizzando l'inverter come punto di accesso (canale 1) per attività di pre-configurazione (come l'autotest e settaggio di parametri ci comunicazione) e analisi e misurazioni di produzione istantanee e periodiche. Il secondo canale Wi-Fi consente la connessione con il proprio router domestico per l'invio di informazioni al cloud Riello Solartech che verranno visualizzate sul portale di supervisione “RS Monitoring”.
Per le impostazioni di installazione e configurazione, utilizzare l'app RS Connect dedicata.

Per il monitoraggio della produzione di energia dell'inverter, utilizzare l'APP RS Connect o registrarsi nel portale cloud RS Monitoring di Riello: www.riello-rsmonitoring.com._"
[Manuale inverter RIELLO PV RS 3.0.pdf](https://github.com/indomus/forum/files/6710522/Manuale.inverter.RIELLO.PV.RS.3.0.pdf)

Non ho ancora trovato la quadra per poter visualizzare (e quindi sfruttare) i dati di produzione dell'impianto...è WiFi ma non riesco a trovare il modo per farlo vedere al sistema :(

Esiste qualche "tool" per riuscire nell'intento? Grazie a tutti!!!

**Comment by Pierluigi2497** [comment]
Non ho ben capito la richiesta, non riesce a visualizzare le informazioni dell'inverter dall'applicazione "RS CONNECT" o vorrebbe un tool esterno che ti permetta di raccogliere le informazioni per poi poterle sfruttare in altro modo?
Nel secondo caso, se le interessa, sto sviluppando uno script che permetta di raccogliere, le informazioni del grafico, la potenza attuale e la temperatura.

**Reply by Cupra1979** [reply]
Buondì Pierluigi,

riesco a monitorare perfettamente i dati dell'inverter sia dall'app "RS CONNECT" che dal portale riello-rsmonitoring.com, ma non ho modo di integrare tali dati in alcuna maniera. Vorrei poterli integrare almeno in HA (produzione istantanea, totale ecc...) per gestire di conseguenza i carichi durante la giornata.

Se si riuscisse a fare qualcosa sarebbe fantastico!

**Reply by Pierluigi2497** [reply]
Ho uno script ancora in fase di sviluppo e testing ma funzionante.
Non so come si potrebbe integrare su HA perché non lo conosco ma è scritto con python quindi molto semplice da sfruttare per altri scopi.
Lo trovi su questo repository https://github.com/Pierluigi2497/Riello-RSTool

**Reply by Cupra1979** [reply]
Grazie, ma purtroppo sono a zero a livello di programmazione/codice/script ecc...
Ci vorrebbe qualcuno in grado di creare una integrazione specifica per Home Assistant...io purtroppo non ne ho le competenze

**Reply by Pierluigi2497** [reply]
Tempo fa diedi un'occhiata ad Home Assistant ma non lo userò per questo progetto, almeno per adesso.
Magari, in futuro, quando deciderò di implementare un'interfaccia grafica potrei aver bisogno di HA, in tal caso aggiornerò il repository di github.

**Comment by AleRo2002** [comment]
Buonasera, sto avendo lo stesso problema. 
Vorrei integrare in home assistant i flussi di energia prodotti dal sistema fotovoltaico Riello.
Hai trovato qualche soluzione?
